README.md

```diff
@@ -1,2 +1,20 @@
 # HW01
-Edit the readme
+## Ariana F. Tortolani 
+### 07/05/2020
```

Comment:
After reading several bios and coding one myself, I had a curious question and I don't know if you noticed it too or perhaps, you know the answer to it. I wonder how exactly does RStudio know the different font size associated to the number of hashtags used when writing the header? Is there a specific 'already set' allocation of the number of hashtags to the size of fonts? If it is not set, how can we change them accordingly?

Reply:
I'm actually not sure what the specifics of it are.  When I looked it up I believe it was the more #'s, the smaller the font size but I didn't know how to change the exact point size of the font.  If you use the same number of #'s on two different lines then those should be the same size (for example my name here and the about me heading below)

Reply:
> After reading several bios and coding one myself, I had a curious question and I don't know if you noticed it too or perhaps, you know the answer to it. I wonder how exactly does RStudio know the different font size associated to the number of hashtags used when writing the header? Is there a specific 'already set' allocation of the number of hashtags to the size of fonts? If it is not set, how can we change them accordingly?

I think that should be something written in RMarkdown? I guess the #'s are read as character strings. When you click "preview" it will automatically count the number of #, and return the value of font size according to there relationship that is previously saved in the algorithm. 
I'm not sure! There could be more advanced ways to achieve this lol